pandas/io/sql.py

```diff
@@ -473,8 +473,9 @@ def read_sql_query(
     --------
     >>> from sqlalchemy import create_engine  # doctest: +SKIP
     >>> engine = create_engine("sqlite:///database.db")  # doctest: +SKIP
+    >>> sql_query = 'SELECT int_column FROM test_data  # doctest: +SKIP
```

Comment:
```suggestion
    >>> sql_query = "SELECT int_column FROM test_data"  # doctest: +SKIP
```

Reply:
Thanks @mroeschke :+1: I do this to fast in the web IDE :sweat: